zppy/templates/e3sm_diags.bash

```diff
@@ -236,7 +236,7 @@ from e3sm_diags.parameter.tc_analysis_parameter import TCAnalysisParameter
 from e3sm_diags.run import runner
 
 short_name = '${short}'
-test_ts = '${ts_dir_destination}'
+test_ts = '${ts_dir_primary}'
```

Comment:
Without this change, the `mvm` test actually wasn't generating plots. Multiple errors were going unnoticed, including:
```
IndexError: list index out of range
RuntimeError: Invalid year range specified for test/reference time series data: end_year=1853>1851=var_end_yr
RuntimeError: Requested years are outside of available sst obs records.
IndexError: index 0 is out of bounds for axis 0 with size 0
```